src/main/java/eu/ai4work/sws/service/RuleEngineService.java

```diff
@@ -50,16 +53,17 @@ public SlidingDecisionResult applySlidingDecisionRules(int noOfTrucksInQueue, in
      * @return FIS object.
      * @throws Exception if the FCL file cannot be found or parsed.
      */
-    private FIS initializeFuzzyInferenceSystem() throws Exception {
+    private FIS initializeFuzzyInferenceSystem(String fclRulesFilePath) throws Exception {
         try {
-            URL fuzzyLogicRulesResourceUrl = getClass().getClassLoader().getResource(TRUCK_SCHEDULING_RULES_FILE);
+            URL fuzzyLogicRulesResourceUrl = getClass().getClassLoader().getResource(fclRulesFilePath);
             if (fuzzyLogicRulesResourceUrl == null) {
-                throw new Exception("Fuzzy Control Language (FCL) file not found: " + TRUCK_SCHEDULING_RULES_FILE);
+                throw new Exception("Fuzzy Control Language (FCL) file not found: " + fclRulesFilePath);
             }
             FIS fuzzyInferenceSystem = FIS.load(fuzzyLogicRulesResourceUrl.getPath(), false); // verbose set to 'false' because to avoid GUI-related processing
             if (fuzzyInferenceSystem == null) {
-                throw new Exception("Failed to initialize Fuzzy Control Language (FCL) file: " + TRUCK_SCHEDULING_RULES_FILE);
+                throw new Exception("Failed to initialize Fuzzy Control Language (FCL) file: " + fclRulesFilePath);
             }
+            logger.info("Successfully initialized FIS from file: {}", fclRulesFilePath);
```

Comment:
As this will be logged for each call to the service and is not important in normal operation mode, I would suggest to set this to log level `debug`.